src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/domain/BlogPostEntityListener.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers.blog_posts.domain;
+
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+
+import org.springframework.cache.CacheManager;
+
+import pl.tomaszdziurko.jvm_bloggers.SpringApplicationContext;
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.AggregatedRssFeedProducer;
+
+public class BlogPostEntityListener {
+
+    @PostPersist
+    @PostUpdate
+    @PostRemove
+    public void invalidateRssCache(BlogPost blogPost) {
+        getCacheManager().getCache(AggregatedRssFeedProducer.RSS_CACHE).clear();
+    }
+
+    private CacheManager getCacheManager() {
+        return SpringApplicationContext.getBean(CacheManager.class);
```

Comment:
I'm afraid it's not easily possible at the moment: https://github.com/spring-projects/spring-boot/issues/739

**TLDR;**
Having _Spring Boot_ application packaged into a single distributable uber jar will require distributing it with additional externally provided dependency (aspectj.jar) and setting load-time weaving by adding `-javaagent` path to AspectJ weaving agent to startup script in order to get `@Configurable` annotation working. It will also complicate development and testing environment as it will require additional development environment setup.